### ISSUE TITLE
Alert settings turn themselves off?

### ISSUE THREAD

**Issue by therentabrain** [PhantomBot/PhantomBot#2628]
PhantomBot Version: 3.4.7 (updating today)
O/S Version: Windows

Detailed Description:
I am about to update PhantomBot, but I don't see this kind of thing addressed in the changes, so I'm assuming it's still a problem...

Three times, spread out over weeks, I have wondered what happened to certain alerts.  I log into the dashboard using Chrome, go to Alerts, and discover that the alert (Hosting, for example) is turned off.  I turn it back on.

Unfortunately I haven't kept track of which alerts have turned themselves off.  I know today it was Hosting, but I am pretty sure it was another kind of alert last time.

I'm a very careful user and I'm fairly certain I didn't turn off the alerts.  That said, maybe there should be some kind of Confirmation or Submit button or something, because I can imagine (especially with a touchscreen, which I wasn't using) it would be easy to accidentally flip one.

Any additional info I can possibly provide to help?  Anyone else experienced this?

Thanks all :)

**Comment by gmt2001** [comment]
Host alerts currently auto-disable if the special IRC connection for them is rejected

**Comment by therentabrain** [comment]
Confirmed, it just happened again.  Is there a fix planned?  When the connection reconnects, it should ideally resume the alerts, right?  Thanks :)

**Comment by gmt2001** [comment]
The fix is to get the correct OAuth setup. It notifies you in console when it is bad

**Comment by gmt2001** [comment]
Improvements to how things like this are handled are always on the roadmap but I currently have 89 tickets open across 2 ticket trackers plus 115 tickets on the exception tracker. I will queue this up but I don't know how long it will take to resolve.

**Comment by therentabrain** [comment]
Yeah, I understand, I appreciate PhantomBot a lot.  Would it help if I donated a little?  I know donations of time can be more valuable, and I'm into trying to help fix things, but I'm new at this.

I tried to do proper OAuth and I thought it was working but then it broke a few weeks ago and said it had expired.  Is it possible I have OAuth working properly for the broadcaster but not for the bot account (or vice versa)?  I've been doing tech for decades so if I can figure out what went wrong for me, I would be happy to try to improve the documentation or figure out if there's a problem in a script.  But I don't know enough to figure it out.  :)

It's been a while since I tried, so I'm a little rusty on what I was seeing at the time... but one question I had:  Is there documentation of all the botlogin variables and what they mean?  I am confused by webauth vs webauthro, apirefresh, oauth vs apioauth.  I feel like if I can get the correct values in there, and remove whatever is still doing the deprecated method, it would be fine.  

Thanks :)
